unit_tests/test_wait.py

```diff
@@ -223,7 +223,7 @@ def test_wait_for_log_timeout_when_no_start_line(self, tmp_path):
         t.daemon = True
         file_path.touch()
         with pytest.raises(TimeoutError, match="Timeout occurred while waiting for start log line"), \
-                wait_for_log_lines(node=node, start_line_patterns=["start"], end_line_patterns=["end"], start_timeout=0.3, end_timeout=1):
+                wait_for_log_lines(node=node, start_line_patterns=["start"], end_line_patterns=["end"], start_timeout=0.5, end_timeout=1.5):
```

Comment:
Need just to mock the `time.sleep` per a unit test.
It is not needed at all to wait for real time intervals.

Moreover, we should not change time values just to make unit tests work.
If it was needed to have `0.1s` sleep, then usage of a mock should completely solve the problem.

Reply:
fixed